tests/unit_tests/test_matchmaker_queue_algorithm.py

```diff
@@ -152,6 +152,7 @@ def test_match_graph_will_not_include_matches_below_threshold_quality(player_fac
     }
 
 
+@pytest.mark.filterwarnings("ignore:.*'caplog' fixture")
```

Comment:
Maybe, you could make ticket To solve this warnings.

Reply:
There isn't anything to fix. We just ignore the warning because we know calling `set_log_level` multiple times is fine. https://github.com/HypothesisWorks/hypothesis/issues/377

Reply:
Ok, np. I'm thinking if we should leave the link in a comment on this line? `# https://github.com/HypothesisWorks/hypothesis/issues/377`?

Reply:
Yea that’s a good idea